src/main/java/COSE/OneKey.java

```diff
@@ -483,22 +496,37 @@ static public OneKey generateKey(AlgorithmID algorithm) throws CoseException {
      * @throws CoseException
      */
     static public OneKey generateKey(AlgorithmID algorithm, String parameters) throws CoseException {
+        return generateKey(algorithm, null, null);
```

Comment:
Looks like this should be `generateKey(algorithm, parameters, null)`?

Reply:
Good catch